includes/class-wc-gateway-ppec-client.php

```diff
@@ -261,7 +261,7 @@ public function get_set_express_checkout_params( array $args ) {
 			$params['USERSELECTEDFUNDINGSOURCE'] = 'Finance';
 		}
 
-		if ( 'checkout' === $args['start_from'] ) {
+		if ( ! $args['skip_checkout'] ) {
```

Comment:
Nitpick: A comment here explaining what does `ADDROVERRIDE` mean would be quite useful (or just a comment in the function header pointing to the API docs)